spec/ParsePolygon.spec.js

```diff
@@ -129,7 +129,7 @@ describe('Parse.Polygon testing', () => {
   });
 
   it('polygonContain query', (done) => {
-    const points1 = [[0,0],[0,1],[1,1],[1,0]];
```

Comment:
This is the most important change. If the coordinates of either the polygon or the coordinate we are seeing if is contained within the polygon are reversed from what we intend, the check will fail.